shared/src/main/scala/NumberHelpers.scala

```diff
@@ -161,18 +161,46 @@ object NumberHelpers:
       case l: VList  => VList.from(temp)
 
   def toBaseDigits(value: VNum, base: VNum): VList =
+    if base == VNum(-1) then
```

Comment:
What if `base` is `VNum.complex(0, -1)`? Or `VNum.complex(-1, -1)`?

Reply:
val real = compToBase(value.real, base.real)
    val imag = compToBase(value.imag, base.imag)

Look at that and tell me how `base` is gonna be `VNum.complex` given that `base.real` and `base.imag` both return `Real`s.